torch/csrc/jit/frontend/tracer.cpp

```diff
@@ -894,6 +894,27 @@ autograd::Variable getSizeOf(const autograd::Variable& var, int64_t dim) {
   return size_var;
 }
 
+autograd::Variable getNumelOf(const autograd::Variable& var) {
```

Comment:
Could we refactor this with `getSizeOf` maybe ?

Reply:
I feel it would add more complexity for handling different arguments based on operator types, while currently we only need inserting a constant in graph for `dim` of `Size`. 
If we choose to address arguments outside of the function, then we'd need the same boilerplate for getting graph from tracing state. 
Please let me know if you have a better idea.

Reply:
yea it's fine as is